tools/open_dataset_tool/objects365.py

```diff
@@ -0,0 +1,194 @@
+    "Lettuce": 260, "Garlic": 261, "Swan": 262, "Helicopter": 263, "Green Onion": 264, "Sandwich": 265, "Nuts": 266, "Speed Limit Sign": 267, "Induction Cooker": 268, "Broom": 269,
+    "Trombone": 270, "Plum": 271, "Rickshaw": 272, "Goldfish": 273, "Kiwi fruit": 274, "Router/modem": 275, "Poker Card": 276, "Toaster": 277, "Shrimp": 278, "Sushi": 279,
+    "Cheese": 280, "Notepaper": 281, "Cherry": 282, "Pliers": 283, "CD": 284, "Pasta": 285, "Hammer": 286, "Cue": 287, "Avocado": 288, "Hamimelon": 289,
+    "Flask": 290, "Mushroom": 291, "Screwdriver": 292, "Soap": 293, "Recorder": 294, "Bear": 295, "Eggplant": 296, "Board Eraser": 297, "Coconut": 298, "Tape Measure/Ruler": 299,
+    "Pig": 300, "Showerhead": 301, "Globe": 302, "Chips": 303, "Steak": 304, "Crosswalk Sign": 305, "Stapler": 306, "Camel": 307, "Formula 1": 308, "Pomegranate": 309,
+    "Dishwasher": 310, "Crab": 311, "Hoverboard": 312, "Meat ball": 313, "Rice Cooker": 314, "Tuba": 315, "Calculator": 316, "Papaya": 317, "Antelope": 318, "Parrot": 319,
+    "Seal": 320, "Butterfly": 321, "Dumbbell": 322, "Donkey": 323, "Lion": 324, "Urinal": 325, "Dolphin": 326, "Electric Drill": 327, "Hair Dryer": 328, "Egg tart": 329,
+    "Jellyfish": 330, "Treadmill": 331, "Lighter": 332, "Grapefruit": 333, "Game board": 334, "Mop": 335, "Radish": 336, "Baozi": 337, "Target": 338, "French": 339,
+    "Spring Rolls": 340, "Monkey": 341, "Rabbit": 342, "Pencil Case": 343, "Yak": 344, "Red Cabbage": 345, "Binoculars": 346, "Asparagus": 347, "Barbell": 348, "Scallop": 349,
+    "Noddles": 350, "Comb": 351, "Dumpling": 352, "Oyster": 353, "Table Tennis paddle": 354, "Cosmetics Brush/Eyeliner Pencil": 355, "Chainsaw": 356, "Eraser": 357, "Lobster": 358, "Durian": 359,
+    "Okra": 360, "Lipstick": 361, "Cosmetics Mirror": 362, "Curling": 363, "Table Tennis": 364
+}
+CLASS365_LABEL_TO_NAME = {val: key for key, val in CLASS365_NAME_TO_LABEL.items()}
+
+def txtywh2cxcywh(top_left_x, top_left_y, width, height):
+    cx = top_left_x + (width / 2)
+    cy = top_left_y + (height / 2)
+    w = width
+    h = height
+    return cx, cy, w, h
+
+
+def cxcywh2cxcywhn(cx, cy, w, h, img_w, img_h):
+    return cx / img_w, cy / img_h, w / img_w, h / img_h
+
+def download_file(url, path, max_retries=30, delay=5, silent=True):
+    silent_option = "sS" if silent else ""
+    for attempt in range(max_retries):
+        try:
+            subprocess.run([
+                "curl", "-#", f"-{silent_option}L", "-o", str(path),
+                "--connect-timeout", "30",
+                "--max-time", "300",
+                "-C", "-",
+                url
+            ], check=True)
+            print(f"Successfully downloaded {url} to {path}")
+            return
+        except subprocess.CalledProcessError as e:
+            print(f"Attempt {attempt + 1} failed to download {url}: {e}")
+            if attempt < max_retries - 1:
+                time.sleep(delay)
```

Comment:
Can you add retry log more explicitly? For example, 

```suggestion
        except subprocess.CalledProcessError as e:
            print(f"Attempt {attempt + 1} failed to download {url}: {e}")
            print(f"Retry download after {delay} seconds ...")
            if attempt < max_retries - 1:
                time.sleep(delay)
```